input/_Navbar.cshtml

```diff
@@ -4,8 +4,8 @@
         Tuple.Create("Blog", Context.GetLink("blog")),
         Tuple.Create("Community", Context.GetLink("community")),
         Tuple.Create("Documentation", Context.GetLink("docs")),
-        Tuple.Create("API", Context.GetLink("api")),
```

Comment:
Revert these as well, I have them here for future use, mapping them to docs would be incorrect. If you want to comment them out so that there are no dead links, that's ok.

Reply:
Ok

Reply:
These are still getting redirected to /docs